### PR TITLE
adds policy context section

### PR DESCRIPTION
**Pull request by kccrtv**
# Description and Motivation
This PR introduces a new section to the HET app, implementing the initial mockup developed on Webflow. This addition provides a comprehensive policy context on the gun violence epidemic, helping users connect the data with real-world community efforts addressing this critical health issue. The framework is designed to be scalable, allowing for the inclusion of additional policy context write-ups in the future, ensuring that the platform continues to evolve and address emerging health equity challenges.

View updates at https://deploy-preview-3503--health-equity-tracker.netlify.app/policy

## Has this been tested? How?
tests passing

## Screenshots (if appropriate)
Webflow Mockup:

> Screenshot 2024-07-30 at 8 55 27 AM

Context Section addition:

> Screenshot 2024-07-30 at 9 03 11 AM
> Screenshot 2024-07-30 at 9 03 45 AM

## Types of changes
- New content or feature

### CODE REVIEW

File: frontend/src/pages/Landing/OverviewTab.tsx
Comment (eriwarr) [inline frontend/src/pages/Landing/OverviewTab.tsx:1]: Did you mean to check this file in?

File: frontend/src/pages/Landing/ContextSection.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import ContextSectionTabs from "./ContextSectionTabs";
+
+export default function ContextSection() {
+	return (
+		<>
+			<section className='py-4 xs:px-8 md:px-24 lg:px-80 max-w-screen'>
+				<p  className='my-3 text-center font-roboto text-smallest font-semibold uppercase text-black'>
+					In Focus
+				</p>
+				<div className='flex w-full flex-col justify-center items-center md:block sm:block'>
```
Comment (eriwarr) [inline frontend/src/pages/Landing/ContextSection.tsx:10]: You should be able to leave off the `sm:block` and `md:block `properties.

File: frontend/src/pages/Landing/ContextSectionTabs.tsx
Comment (eriwarr) [inline frontend/src/pages/Landing/ContextSectionTabs.tsx:1]: Is this file a part of this PR? I tried interacting with it but couldn't get it to work. 
Screenshot 2024-07-30 at 11 28 24 AM

File: frontend/src/pages/Landing/OverviewTab.tsx
Comment (kccrtv) [reply frontend/src/pages/Landing/OverviewTab.tsx:1]: removed -- will refactor using buttons like the methodology nav menu

File: frontend/src/pages/Landing/ContextSectionTabs.tsx
Comment (kccrtv) [reply frontend/src/pages/Landing/ContextSectionTabs.tsx:1]: removed -- will refactor using buttons like the methodology nav menu

**Review by benhammondmusic — APPROVED** [review]
looks great; code is clean and easy to follow. thanks!

**Review by eriwarr — APPROVED** [review]
Looks great Kim!

**Pull request merged**